README.md

```diff
@@ -1 +1,247 @@
+<img src="logo.png" width="100px" style="float:right; margin-top:-45px"/>
+
+**Author:** Tek Raj Chhetri | <tekraj@mit.edu>
+
+## Overview
+_BrainKB serves as a knowledge base platform that provides scientists worldwide with tools for searching, exploring, and visualizing Neuroscience knowledge represented by knowledge graphs (KGs)_. Moreover, BrainKB provides cutting-edge tools that enable scientists to contribute new information (or knowledge) to the platform, ensuring it remains the go-to destination for all neuroscience-related research needs.
+
+The main objective of BrainKB is to represent Neuroscience knowledge as a knowledge graph such that it can be used for different downstream tasks, such as making predictions and new inferences in addition to querying and viewing information.  The expected outcome of the BrainKB includes the following:
+
+- (Semi-)Automated extraction of Neuroscience knowledge from structured, semi-structured, and unstructured sources, representing the extracted knowledge via KGs.
+ 
+- Visualization of the KGs.
+
+- Provides features to perform different analytics operations over the BrainKB KGs.
+- (Semi-)automatically validates (e.g., quality) the extracted KGs to ensure that the BrainKB KGs are high quality. 
+- Provides the ability to ingest data in batch or streaming mode for the automated extraction of KGs.
+
+## Why BrainKB?
+
+- **Limited Availability of Platforms for Integrating Neuroscience Data into Knowledge Graphs:** In fields such as biomedicine, many platforms exist, such as, [SPOKE](https://doi.org/10.1093/bioinformatics/btad080) and [CIViC](https://civicdb.org/welcome). <span style="color: red;">However, such resources are comparatively limited in the domain of neuroscience.</span> [LinkRBrain](https://doi.org/10.1016/j.jneumeth.2014.12.008), a web-based platform that integrates anatomical, functional, and genetic knowledge, is among the limited number of such resources. [BrainKnow](http://www.brain-knowledge-engine.org/), the most recent platform, is another platform that is designed to [synthesizes and integrates neuroscience knowledge from scientific literature](https://arxiv.org/pdf/2403.04346.pdf). Additionally, projects like [DANDI](https://dandiarchive.org/) are making strides by enabling sharing of neurophysiology data together with its metadata. 
+
+- **Lack of Support for Heterogeneous Data Sources:** The current platforms in neuroscience are limited in their ability to handle a diverse range of data sources. For instance, [LinkRBrain](https://doi.org/10.1016/j.jneumeth.2014.12.008) can only integrate knowledge from 41 databases, whereas [BrainKnow](http://www.brain-knowledge-engine.org/) solely focuses on scientific literature. <span style="color: red;">However, knowledge is not restricted to just databases or scientific literature, and there is a need for platforms that can accommodate a wider variety of sources (e.g., structured, semi-structured and unstructured sources).</span>
+
```

Comment:
I think that above overview would be very incomplete without mentioning/comparing to efforts in EU/Ebrains:
- https://github.com/openMetadataInitiative for metadata harmonization
- their "knowledge graph" : https://www.ebrains.eu/data/find-data/find-data/

Reply:
Updated!